pegasus/sites.v3/code.org/public/privacy.redirect

```diff
@@ -0,0 +1 @@
+/privacy/index
```

Comment:
I tried doing this with just `/privacy` but was getting these errors: 

> inline template:81: warning: string literal in condition
inline template:95: warning: string literal in condition